sql/synth_recipes.sql

```diff
@@ -60,55 +60,6 @@ CREATE TABLE `synth_recipes` (
 -- Dumping data for table `synth_recipes`
 --
 
-DELIMITER $$
```

Comment:
Add back in the -DELIMITER $$ as a Comment or as is posted on line 63 there? Sorry I am trying to figure this out from just 1 or two learning sessions of how github works.